pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
OK. The pkg naming convention is not using underscore in go. 
Please check https://golang.org/doc/effective_go.html#package-names.

I cannot see why you add the `_test` prefix. Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary. The main reason we want to put it into a different pkg is to avoid circle import I think.